### PR TITLE
ocm: Cut release-2.4 branch for console-api

### PR DESCRIPTION
**Pull request by KevinFCormier**
open-cluster-management/backlog#14309

### CODE REVIEW

File: ci-operator/config/open-cluster-management/console-api/open-cluster-management-console-api-release-2.4.yaml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    export SELF="make -f Makefile.prow"
+    make -f Makefile.prow sonar/js/jest-init
+    make -f Makefile.prow unit-tests
+    make -f Makefile.prow sonar/js/prow
+  container:
+    from: src
+  postsubmit: true
+  secrets:
+  - mount_path: /etc/sonarcloud/
+    name: acm-sonarcloud-token
```
Comment (KevinFCormier) [inline ci-operator/config/open-cluster-management/console-api/open-cluster-management-console-api-release-2.4.yaml:70]: @ray-harris Following https://github.com/open-cluster-management/cicd-docs/blob/main/prow/new_release.md, there was already a `sonarcloud` test here, but not as a postsubmit, so I added `sonar-post-submit` as well. Does this look right?

**Review by birsanv — APPROVED** [review]
lgtm

File: ci-operator/config/open-cluster-management/console-api/open-cluster-management-console-api-release-2.4.yaml (hunk 1 above)
Comment (ray-harris) [reply ci-operator/config/open-cluster-management/console-api/open-cluster-management-console-api-release-2.4.yaml:70]: Yes

**Comment by ray-harris** [comment]
Looks good to me as well.

/unhold

**Review by chenz4027 — APPROVED** [review]
lgtm - thanks Kevin!

**Pull request merged**